src/System.Collections.NonGeneric/tests/Performance/Perf.HashTable.cs

```diff
@@ -48,16 +52,20 @@ public void ctor()
         public void GetItem(Hashtable table)
         {
             object result;
-            string key = PerfUtils.CreateString(50);
+            PerfUtils utils = new PerfUtils(983452);
```

Comment:
Otherwise it will use the same seed that is used in the generation of the collections used as MemberData which will cause the key being generated to already exist within the Hashtable